### ISSUE TITLE
@types/simplebar-react 

### ISSUE THREAD

**Issue by Riddeller** [Grsmto/simplebar#180]
Hi,

We have types for simplebar but not for the react implementation. I've tried writing type definition file but I can't get it to work.

Does anyone have working types for react implmentation?

**Comment by Riddeller** [comment]
Anyone looking for a quick fix until this exists you can just create new defintion with the contents of:

`
// Type definitions for react-simplebar using simplebar.js 2.4
// Project: https://github.com/Grsmto/simplebar
// Definitions by: Gregor Woiwode <https://github.com/gregonnet>, Leonard Thieu <https://github.com/leonard-thieu>
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
// TypeScript Version: 2.1

import React = require("react");
export as namespace SimpleBar;
export default class SimpleBar extends React.Component<any, any> {}
`

**Comment by Grsmto** [comment]
Hi,
I'm happy to provide something to support TS types but I'm not using neither TS or Flow on this project yet so I probably won't do it myself.

**Comment by Grsmto** [comment]
I don't have time to do a PR to definitely typed but if someone has, here is a basic definition:

```
declare module 'simplebar-react' {
  import { FC } from 'react';

  const SimpleBar: FC;

  export default SimpleBar;
}
```

**Comment by AndreyKondakov** [comment]
<SimpleBar style={{ height: '400px' }} data-simplebar-force-visible>
Hey. Can you please tell me. How can I change the settings so that the scrolls are always displayed in react?

**Comment by Grsmto** [comment]
Closed in #346.
